### ISSUE TITLE
Accessibility [MIGHT NEED TO BECOME SEVERAL DISTINCT GITHUB ISSUES...]

### ISSUE THREAD

**Issue by liaprins-czi** [napari/product-heuristics-2020#11]
There are several color contrast and text size issues that are not WCAG (web content accessibility guidelines) compliant. While napari is not "web content" we should be aware of presenting tools that can be ready by individuals with sight differences.

Light theme layer selection color is strange (yellow) and inaccessible with text. When toggle the theme (View > Toggle theme) the background of most of the viewer changes to be light, but not the background of the layer controls + list, even when they are part of the viewer window. Same for when I hover on the little line above the layer control panel or layer list panel when within the viewer window: upon hover, the background behind the line turns from light to dark abruptly. Also sometimes one panel is light colored and another is not, but I could not discern how to reproduce this.
__________________________________________________________
**Recommendation**

Will need to check contrast and size standards. There are lots of tools available to check web content that we may be able to mimic: (https://www.digitala11y.com/color-contrast-analyzers-checkers/ and https://www.w3.org/WAI/ER/tools/ to start).

The colors used for light theme in general should be more carefully chosen. Wherever the background is dark when in dark mode, it should correspondingly be light when in light mode.

Moderate issue
●●●   High design work
▼▼▼   High development work

When toggle the theme (View > Toggle theme) the background of most of the viewer changes to be light, but not the background of the layer controls + list, even when they are part of the viewer window:
![lightmode](https://user-images.githubusercontent.com/53277128/95918805-4ffe8080-0d61-11eb-9ef9-eb955e217b65.png)

Light theme layer selection color is strange (yellow):
![lightmode2](https://user-images.githubusercontent.com/53277128/95918886-70c6d600-0d61-11eb-8194-ae4f2ed30439.png)

Yellow color in light mode is not accessible:
![a11yyellow](https://user-images.githubusercontent.com/53277128/95920356-22ff9d00-0d64-11eb-9f48-8cbbddb1a4e4.png)

**Comment by goanpeca** [comment]
What is the status of this issue @liaprins-czi ?

Do we plan to open more issues for specific things to fix, and keep and overall list of the different problems we will need to solve?

Thanks!

**Comment by liaprins-czi** [comment]
@goanpeca This should maybe not even be an issue at all, haha. I think we are planning to do an accessibility audit of napari viewer sometime this half (or have one done...), then we'd create GH issues for each of those found. I will close this issue in the meantime, as we have the need for an a11y audit currently documented in our team OKRs.
